documentation/docs/guidedtour.md

```diff
@@ -0,0 +1,188 @@
+```
+
+When the files are downloaded into the current directory, launch the Jenkins server by using the following command:
+
+
+```sh
+./cx-server start
+```
+
+For more information on the Jenkins lifecycle management and how to customize your Jenkins, have a look at the [Operations Guide for Cx Server][devops-docker-images-cxs-guide].
+
+
+## Create Your First Pipeline
+
+For the beginning, we recommend using an SAP sample application. The repository [cloud-cf-helloworld-nodejs][cloud-cf-helloworld-nodejs] contains a simple `nodejs` application that can be enriched with a pipeline built with MTA and deployed into the Cloud Foundry environment.
+
+1. Fork the [cloud-cf-helloworld-nodejs][cloud-cf-helloworld-nodejs] repository into your GitHub organization.
+
+1. Select the `1_REST_persist_in_Memory` branch of your [cloud-cf-helloworld-nodejs] fork. 
+
+1. Create a new file with the name `Jenkinsfile` in the root level of your repository and enter the following code:
+   ```
+   @Library('piper-lib-os') _
+   node() {
+       stage('prepare') {
+           checkout scm
+           setupCommonPipelineEnvironment script:this
+       }
+   }
+   ```
+   The "prepare" step synchronizes the repository and initializes the project specific settings. For more information about Jenkinsfiles and pipelines, see [Using a Jenkinsfile][jenkins-io-jenkinsfile].
+   
+1. Save your changes to your remote repository.
+
+1. To set up a Jenkins job for your repository, open the Jenkins UI under `http://<jenkins-server-address>:<http-port>` and choose **New Item**. Per default, the `cx-server` starts Jenkins on HTTP port `80`. For more information, see the [Jenkins User Documentation][jenkins-io-documentation].
+   <p align="center">
+   ![Clicke New Item](../images/JenkinsHomeMenu-1.png "Jenkins Home Menu")
+   </p>
+
+1. Provide a name for your new item (for example, *My First Pipeline*) and select **Pipeline**.
+   <p align="center">
+   ![Create Pipeline Job](../images/JenkinsNewItemPipeline-1.png "Jenkins New Item")
+   </p>  
+
+1. For **Definition** in the **Pipeline** options, choose **Pipeline script from SCM**.
+
+1. For **SCM**, choose **Git**.
+
+1. For **Repository URL** in the **Repositories** section, enter the URL of your Git repository, for example `https://github.com/<your-org>/cloud-cf-helloworld-nodejs`. **Note:** If your repository is protected, you must provide your credentials in the **Credentials** section.
+   <p align="center">
+   ![Create Pipeline Job](../images/JenkinsNewItemPipeline-2.png "Jenkins New Item")
+   </p>
+
+1. For **Branch Specifier** in the **Branches to build** section, enter the branch name `*/1_REST_persist_in_Memory`.
+
+1. Choose **Save**.
+
+1. To run your pipeline, choose **Build Now** in the job UI.
+
+
+## Add a Build Step
+
+1. In your `Jenkinsfile`, add the following code snippet after `stage("prepare") { ... }`:
```

Comment:
Maybe we should explain where the snippet should be added. It needs to be after the build stage, but inside the node closure.

Reply:
I'd say, that's the learning. If the reader add it to the wrong location and despairs, he could compare to the full sample at the end.
The challenge is low and reasonable.